pos_disable_product_attribute/models/product_attribute.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2023-Today
+# @author Emanuel Cino
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, fields, models
+
+
+class ProductAttribute(models.Model):
+    _inherit = "product.attribute"
+
+    pos_visibility = fields.Selection(
```

Comment:
Not-blocking: I'd align to the standard `available_in_pos` in products and make this a Boolean, with the same name

But perhaps it's only my opinion here, feel free to ignore